deployment/address_book.go

```diff
@@ -34,9 +34,13 @@ var (
 type TypeAndVersion struct {
 	Type    ContractType
 	Version semver.Version
+	Label   *string
```

Comment:
Why not a map[string]string to support multiple labels?

Reply:
Was just going to say. :D

Reply:
We do want set behavior, though I was thinking map[string]struct{}{}. I hate that there isn't a built-in set.

Reply:
it's a little ugly but I'm now getting something like this 
```
    "0x72929d691f2dc2d897e1f91CdaD0d0C4472dE54F": {
      "Type": "RBACTimelock",
      "Version": "1.0.0",
      "Metadata": {
        "SA": {}
      }
    },
 ```
 but yeah.. it is what it is with go unfortunately